Loop/DerivedAssetsBase.xcassets/glucose.colorset/Contents.json

```diff
@@ -20,7 +20,7 @@
         }
       ],
       "color" : {
-        "color-space" : "display-p3",
+        "color-space" : "srgb",
```

Comment:
yes, the colour values were correct, just the Color-space was incorrect. I double checked them when I changed the space, since as you know, changing the space changed the values.